src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsBuildWrapper.java

```diff
@@ -65,21 +65,24 @@
 
 /**
  * Build wrapper that alters the console so that passwords don't get displayed.
- * 
+ *
  * @author Romain Seguy (http://openromain.blogspot.com)
  */
 public final class MaskPasswordsBuildWrapper extends SimpleBuildWrapper {
 
     private final List<VarPasswordPair> varPasswordPairs;
+    private final List<VarMaskRegex> varMaskRegexes;
 
     @DataBoundConstructor
-    public MaskPasswordsBuildWrapper(List<VarPasswordPair> varPasswordPairs) {
+    public MaskPasswordsBuildWrapper(List<VarPasswordPair> varPasswordPairs, List<VarMaskRegex> varMaskRegexes) {

```

Comment:
Ok, thanks.

I tried to retain the previous constructor as well, but when I do, compilation fails:

```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] error: javax.annotation.processing.FilerException: Attempt to reopen a file for path /home/jantman/manheim/git/github_dot_com/mask-passwords-plugin/target/classes/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsBuildWrapper.stapler
[INFO] 1 error
```

I'm sort of lost on how to fix this. It appears that one class can't have multiple ``@DataBoundConstructor``-annotated constructors?